src/app/shared/abstract-component-loader/abstract-component-loader.component.ts

```diff
@@ -0,0 +1,136 @@
+import { Component, ComponentRef, Input, OnChanges, OnDestroy, OnInit, SimpleChanges, ViewChild, ViewContainerRef } from '@angular/core';
+import { Context } from '../../core/shared/context.model';
+import { ThemeService } from '../theme-support/theme.service';
+import { GenericConstructor } from '../../core/shared/generic-constructor';
+import { hasValue, isNotEmpty } from '../empty.util';
+import { Subscription } from 'rxjs';
+import { DynamicComponentLoaderDirective } from './dynamic-component-loader.directive';
+
+@Component({
+  selector: 'ds-abstract-component-loader',
+  templateUrl: './abstract-component-loader.component.html',
+})
+export abstract class AbstractComponentLoaderComponent<T> implements OnInit, OnChanges, OnDestroy {
```

Comment:
Tiny thing. Can we add a description to this new class?  E.g. how it is used or when it should be extended. That way other developers will understand how to use it for future code.